src/main/java/hw_17/Test3.java

```diff
@@ -0,0 +1,29 @@
+package hw_17;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.time.Duration;
+
+public class Test3 {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://makeup.com.ua/ua/");
+
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(15));
+
+        WebElement makeupClubButton = driver.findElement(By.cssSelector("a.header-top-list__link.bc-about-link"));
+        makeupClubButton.click();
+
+        WebElement videoButton = driver.findElement(By.cssSelector("a.bc-1y2bmtx-item"));
+        videoButton.click();
+
+        String currentUrl = driver.getCurrentUrl();
+        Assert.assertTrue(videoButton.isDisplayed());
```

Comment:
After clicking on videoButton on the 22 row you're navigated to another page view and try to verify videoButton is displayed.
But after navigating the selector is changed. So you need to find another selector to verify videoButton is displayed
![Screenshot_7](https://github.com/user-attachments/assets/ae8cae8f-9876-4d43-951c-50640fdcd9f8)

Reply:
i didn't get it
do i need to open another driver with another link (likehttps://club.makeup.com.ua/video/ ) to find the video selector?

Reply:
ok
on this step you're trying to verify that video button is displayed
![Screenshot_3](https://github.com/user-attachments/assets/c434a804-8f31-4344-bd62-b03566dd946f)

But in fact on this step the selector for video button is changed and it is not already a video button
![Uploading Screenshot_4.png…]()

Reply:
so do i need to delete this? or what should i check to be displayed?